releasenotes/notes/add-grover-primitives-10f81efdba93703d.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+
+    :class:`~.Grover` supports the primitives and can use :class:`~.BaseSampler` to
+    calculate the results. Accordingly, ``quantum instance`` in :class:`~.Grover`
+    is pending deprecation and will be deprecated in a future release.
+
+    Example::
+    .. code-block:: python
+
+      sampler = Sampler(...)
+      problem = AmplificationProblem(...)
+      grover = Grover(sampler)
+      result = grover.amplify(problem)
```

Comment:
Imports missing and code looks incomplete.

Reply:
It's a `code-block` (not a jupyter notebook). I described an abstract code to show how to use it.